pghoard/basebackup/delta.py

```diff
@@ -94,6 +94,10 @@ def progress_callback(progress_step: ProgressStep, progress_data: ProgressMetric
                     persisted_progress.write(self.metrics)
                     self.last_flush_time = time.monotonic()
                     self.metrics.gauge("pghoard.seconds_since_backup_progress_stalled", 0, tags=tags)
+                    self.log.info(
```

Comment:
This could be a very noisy log line if it is being emitted every time we update our progress?

Reply:
I can remove this, but this line logs the elapsed time only once it reaches the PROGRESS_CHECK_INTERVAL, which means it's not continuous for all phases.

Reply:
OK perfect, ignore me :)